### PR TITLE
added logout page

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added logout page styling and added logout link button on header of wishlist and cart page

### CODE REVIEW

**Review by aniketxparihar — APPROVED** [review]

**Review by Sahil25061999 — APPROVED** [review]
There is nothing much to suggest everything looks good, can be merged.

**Comment by gouravm20** [comment]
no issue found ready for deploy

**Review by pariyar07 — COMMENTED** [review]
Go ahead and merge

File: wishlist.html [hunk 1]
```diff
@@ -30,10 +30,9 @@
               <ul class="right row items-center gap-1 text-light">
                 <li><a href="index.html">Home</a></li>
                 <li>
-                    <a href="login.html" class="btn btn-primary">
-                        Login</a>
+                    <a href="logout.html" class="btn btn-primary">
```
Comment (gouravmarch20) [inline wishlist.html:33]: no issue , relative directorary --> ./logout.html is also an option

**Review by gouravmarch20 — COMMENTED** [review]
no issue , ui is looking great

**Pull request merged**